src/main/java/com/faforever/client/player/PlayerService.java

```diff
@@ -144,6 +147,11 @@ public boolean areFriendsInGame(GameBean game) {
         .collect(Collectors.toList());
   }
 
+  public Optional<Image> getCurrentAvatarByPlayerName(String name) {
+    PlayerBean player = playersByName.getOrDefault(name, null);
+    return Optional.ofNullable(avatarService.loadAvatar(player != null ? player.getAvatar() : null));
```

Comment:
```suggestion
    return Optional.ofNullable(player.getAvatar()).map(avatarService::loadAvatar).orElse(null);
```
If you are already using an optional no need to use a tertiary operator just use the map function.